src/pages/costModels/costModel/addSourceWizard.tsx

```diff
@@ -70,6 +72,7 @@ class AddSourceWizardBase extends React.Component<Props, AddSourcesStepState> {
         };
       }, {}) as { [uuid: string]: { selected: boolean; meta: Provider } };
       this.setState({ checked: initChecked });
+      this.setState({ initialChecked: initChecked });
```

Comment:
We don't really care about the initial state. We only need to count the current selections using the `checked` state.

Reply:
We need to compare the initial state to the current state, as the button will only be enabled when the current state differs from the initial state.

Reply:
Yes, but we don't need a separate object for this. My point being that `checked` already maintains the selected state, so let's reuse it. See comments above